app/helpers/autosign_helper.rb

```diff
@@ -1,5 +1,5 @@
 module AutosignHelper
   def autosign_form
-    link_to "New", new_smart_proxy_autosign_path(@proxy) if authorized_for(:controller => "SmartProxies::Autosign", :action => :create, :auth_object => @proxy)
+    button_tag("New", :type => 'button', :class => 'btn btn-primary fr', :data => {:toggle => 'modal', :target =>'#autosignModal'}) if authorized_for(:controller => "SmartProxies::Autosign", :action => :create, :auth_object => @proxy)
```

Comment:
Pushed it down a bit.
I didn't see any shifting when opening the modal, could you add a screen shot?